data/resistance intro.txt

```diff
@@ -0,0 +1,1187 @@
+				accept
+
+
+
+mission "Resistance: Platings"
+	name "Plating Smuggling"
+	description "Bring 20 Interference Platings from Human space to <destination>, to help the Resistance."
+	landing
+	source "Fourth Shadow"
+	destination "Mebla's Portion"
+	to offer
+		has "Resistance: Charity: done"
+		not "assisted heliarch"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch Containment 4-A: offered"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`You drop off the crates full of charity goods, and the spaceport workers start getting them up on vehicles to transport them to the poor.`
+			`	While you're helping them get the crates out of your cargo hold, you go in there one last time for the final crate, but find it's gone, and a small, thin transceiver is on the floor, near where the crate was.`
+			`	You manage to connect it to your ship's control panel, and after a second, a message pops up in your screen.`
+			`	"Greetings. Thank you for helping us with this operation. It is customary for us to transport 'gifts' like this, but this method takes an awful ammount of time. We are aware that humans developed a type of plating that can interfere with scanners. Bring twenty of those to <destination>, if you wish to help our Resistance."`
+			`	The message clears shortly after you finish reading it, and a bit of smoke starts leaving the device. As if it wasn't obvious enough before, the last word of the message blatantly spells out that whoever sent it is member of the Resistance movement against the Heliarchs.`
+			choice
+				`	(Accept the smuggling job.)`
+				`	(Ignore it, it's best to not get involved in this.)`
+					decline
+			`	You dispose of the small piece of equipment, and trace a route to the nearest human outfitter that sells the requested outfits.`
+				accept
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		log "Factions" "Resistance" `The Resistance is a rarely talked about faction within the Coalition, a group that opposes the Heliarch government.`
+		outfit "Interference Plating" -20
+		payment 1000000
+		conversation
+			`You're about to dock as usual, but get a message pop up again, telling you to go to the edge of the city. You follow the instructions, and find yourself in the middle of several dozen warehouses. Another message asks that you unload the crates and bring them to one specific warehouse, as you see that a cargo loader the spaceport workers use has been prepared for you.`
+			`	As you're doing so, you remember that you didn't get paid for this Resistance's 'charity' job, and you weren't told if you would even get paid for this one, so after the seventh crate you consider dropping this job entirely.`
+			`	Disregarding that, you finally get the last of the Interference Platings, and as you fly your ship back to the more convenient landing pads, you see that <payment> have been added to your account. Exactly the price for twenty Interference Platings.`
```

Comment:
I'm actually not sure on the part right after payment, so I kept it as "have" for now.
like, since the payment is 1 million would it not be best to keep it as "have" ? "1000000 credits have been added to your account", as opposed to "1000000 credits has been added to your account" ? I couldn't remember any instance where you're paid 1M in such a way, so idk which way it's done

Reply:
I agree with @Arachi-Lover in this instance.